main.sh

```diff
@@ -75,7 +75,7 @@ function __b3bp_log () {
   local color="${!colorvar:-$color_error}"
   local color_reset="\x1b[0m"
 
-  if [ "${NO_COLOR}" = "true" ] || [[ "${TERM:-}" != "xterm"* ]] || [ -t 1 ]; then
+  if [ "${NO_COLOR:-}" = "true" ] || ([ "${NO_COLOR:-}" != "false" ] && ([[ "${TERM:-}" != "xterm"* ]] || [ ! -t 2 ])); then
```

Comment:
The logic here is getting complex... might it be time for `if ; then; else; fi`?